.evergreen/run-tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+if [ -z "$NODE_VERSION" ]; then
+  echo "NODE_VERSION environment variable must be specified"
+  exit 1
+fi
+
+NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY}/node-artifacts"
+
+if [ "$OS" == "Windows_NT" ]; then
+    export NVM_HOME=`cygpath -w "$NODE_ARTIFACTS_PATH/nvm"`
+    export NVM_SYMLINK=`cygpath -w "$NODE_ARTIFACTS_PATH/bin"`
+    export PATH=`cygpath $NVM_SYMLINK`:`cygpath $NVM_HOME`:$PATH
+    NODE_TEST="test-node-windows"
```

Comment:
Whats the reason for not using `test-node` here?

Reply:
The script `test-node` references `node_modules/.bin`, which doesn't exist on windows. I could probably just change `test-node` to what I put in for `test-node-windows` so I don't need the extra logic; will try that out.

Reply:
Wow that `.` is easy to miss, it is what it is if there's a different directory shape in windows, up to you.